### DISCUSSION TITLE
React Native - Unable to detect login share from auth network

### DISCUSSION THREAD

**Discussion by miroosama** [Web3Auth/Web3Auth#1170] [Help]
Implementing Custom Authentication with own jwt server in React Native. I am receiving inconsistent behavior during login. Login failed previously, then login worked yesterday, and now it has failed again all with the following message:

<img width="346" alt="Screen Shot 2023-01-05 at 8 03 14 PM" src="https://user-images.githubusercontent.com/32473240/210911894-f071a712-5f23-4b83-9ed0-cc96dbcc3792.png">

Code snippet:

<img width="654" alt="Screen Shot 2023-01-05 at 8 40 02 PM" src="https://user-images.githubusercontent.com/32473240/210912384-3d896a63-3805-4633-9c7e-a71b409a72a3.png">

**Comment by Animalmix55** [comment]
I too am experiencing this issue. It has been persistent for the last day, but I have seen it intermittently

**Comment by chaitanyapotti** [comment]
One of our partner endpoints went down for testnet. Services are back up and running now

**Reply by szmizorsz** [reply]
@chaitanyapotti I am facing the same issue on testnet with email and password login through Auth0. However, Twitter login through Auth0 works fine. Both are consistent, the first never let me log in, and the second always succeeds. 
The error is always "Unable to detect login share from the Auth Network.", even if I try it over a high-speed connection.
Could it be caused by the outage of partner endpoints?

**Reply by szmizorsz** [reply]
My problem was a bad configuration in the login provider configuration, the correct one:

`    const web3authProvider = await web3auth.connectTo(
      WALLET_ADAPTERS.OPENLOGIN,
      {
        loginProvider: 'email',
        extraLoginOptions: {
          domain: auth0DomainForEmailLogin,
          verifierIdField: 'email',
        },
      },
    );
`

Instead of 'email' in the 'verifierIdField' there was 'sub'

**Comment by PrajTS** [comment]
Hi, I'm facing the same issue on testnet for SMS login through Auth). However, Google login through auth0 works fine.
